test/types/tuple/homog/tuple-out-of-bounds-access.good

```diff
@@ -0,0 +1,3 @@
+0
+22
+tuple-out-of-bounds-access.chpl:8: error: halt reached - tuple access out of bounds: 23 (accessing 23*int(64))
```

Comment:
How about something that indicates that the tuple is zero-indexed?

```
tuple-out-of-bounds-access.chpl:8: error: halt reached - tuple access out of bounds: 23 (when accessing a tuple '23*int(64)', allowed indices are 0 through 22)
```

Reply:
How about we make it look more similar to the array OOB access error?

```
aa.chpl:3: error: halt reached - array index out of bounds
note: index was 23 but array bounds are 1..10
```

-> 

```
aa.chpl:3: error: halt reached - tuple index out of bounds
note: index was 23 but the tuple type 23*int(64) has bounds 0..22
```

Reply:
That sounds good to me.